### DISCUSSION TITLE
Is there a wheel for Python 3.13.0 MacOS 14

### DISCUSSION THREAD

**Discussion by greywidget** [psycopg/psycopg#951] [Q&A] [answered]
Hi, 
I'm getting an error on my MacBook M2 trying to run `uv add 'pscopg[binary]'` which claims there is no source distribution or wheel for the current platform (for `psycopg-binary==3.2.3`)

My virtual environment is Python 3.13.0 and my MacOS is Sonoma 14.7.1

Looking at the binary distributions I see:

`psycopg_binary-3.2.3-cp313-cp313-macosx_14_0_arm64.whl` 

which I thought should work. 

Forgive my ignorance - is this the correct wheel for me?
Prior to attempting this upgrade I've been able to install and run an older version `uv add 'psycopg[binary]==3.1.19` on both Ventura and now on  Sonoma.

Thanks kindly

**Comment by dvarrazzo** [comment, marked as answer]
There is no source package for `psycopg-binary`. The OS you mention seems supported. In 3.1.20 we had to drop build for some platforms, as explained [here](https://github.com/psycopg/psycopg/issues/858#issuecomment-2217172495) but it doesn't seem your system should have been affected.

I can't debug your laptop. You should check on your own side why the package you have found available is not installed on it.

**Reply by greywidget** [reply]
Thanks for taking the time and trouble to respond, and also for all of your hard work maintaining this package.

I was just about to start some trouble shooting on my Mac to see if the package would install in isolation or directly via pip (instead of uv) but the problem seems resolved now. I think the only thing that has changed (from my perspective) is uv, which as I'm sure you know is moving fast. 

So my best guess is that something in uv 0.5.1 resolved the problem.

In any case - sorry for having wasted your time and thanks again.

**Reply by dvarrazzo** [reply]
`uv` is very promising, so thank you for testing with it! :smile:
